planning/behavior_path_planner/autoware_behavior_path_lane_change_module/include/autoware/behavior_path_lane_change_module/scene.hpp

```diff
@@ -209,7 +211,7 @@ class NormalLaneChange : public LaneChangeBase
 
   std::pair<double, double> calcCurrentMinMaxAcceleration() const;
 
-  void setStopPose(const Pose & stop_pose);
+  void set_stop_pose(const double arc_length_to_stop_pose, PathWithLaneId & path);
```

Comment:
can this function be `const` ?

Reply:
No, the function modifies member variable